src/tests/integration/test_kontrol_cse.py

```diff
@@ -48,6 +48,10 @@ def test_foundry_dependency_automated(
     if bug_report is not None:
         server._populate_bug_report(bug_report)
 
+    run_constructor = True
+    if test_id in ['CallableStorageTest.test_str()', 'CallableStorageContract.str()']:
```

Comment:
Would `CallableStorageContract.init` fail if we run it with `--run-constructor`? Would it be sufficient to handle the engagement contracts @PetarMax?

Reply:
We don't want to handle this. We need the constraints in constructors as well, either as part of this PR or the next one.